src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -331,3 +331,16 @@ def load_arguments(self, _):
     with self.argument_context('monitor activity-log alert scope remove') as c:
         c.argument('scopes', options_list=['--scope', '-s'], nargs='+')
     # endregion
+
+    # region Log Analytics Workspace
+    with self.argument_context('monitor log-analytics workspace') as c:
+        from azure.mgmt.loganalytics.models import SkuNameEnum
+        c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('workspace_name', options_list=['--workspace-name', '-n'], help="Name of the Log Analytics Workspace.")
+        c.argument('sku', deprecate_info=c.deprecate(hide=True), arg_type=get_enum_type([SkuNameEnum.per_gb2018.value]), help="The SKU of the workspace.", default=SkuNameEnum.per_gb2018.value)
+        c.argument('retention_time', help="The workspace data retention in days.", type=int, default=30)
+
+    with self.argument_context('monitor log-analytics workspace pack') as c:
+        c.argument('intelligence_pack_name', options_list=['--name', '-n'])
+        c.argument('workspace_name', options_list='--workspace-name')
```

Comment:
Why do you add a options_list with the same name of your argument?

Reply:
fix

Reply:
So I tried to change back